stable/mlrun/values.yaml

```
@@ -124,8 +124,8 @@ api:
 
   priorityClassName: ""
 
-  podSecurityContext: {}
-  # fsGroup: 2000
+  podSecurityContext:
+    runAsUser: 1000
```

maybe add group id as well so that the pod doesn't get root permissions from the group
```suggestion
    runAsUser: 1000
    runAsGroup: 3000
```

---

id leave empty and let user decide

---

removed